### PR TITLE
Fix balutil handling of DisplayInternalUI

### PR DESCRIPTION
**Pull request by rseanhall**
Virtually all changes to parsing BootstrapperApplicationData.xml need to be done in mbacore and balutil.

Also, fix Intellisense for dutil and typos/style in thmutil.

### CODE REVIEW

File: src/libs/dutil/thmutil.cpp [hunk 1]
```diff
@@ -492,7 +492,7 @@ DAPI_(HRESULT) ThemeLoadControls(
         case THEME_CONTROL_TYPE_PROGRESSBAR:
             if (pControl->hImage || (pTheme->hImage && 0 <= pControl->nSourceX && 0 <= pControl->nSourceY))
             {
-                wzWindowClass = WC_STATICW; // no such thing as an owner drawn progress bar so we'll make our own out of a static control.
+                wzWindowClass = WC_STATICW; // No such thing as an owner drawn progress bar so we'll make our own out of a static control.
```
Comment (robmen) [inline src/libs/dutil/thmutil.cpp:495]: When comments are on the same line, they are generally considered part of the sentence, thus not capitalized.

File: src/libs/dutil/dutil.vcxproj [hunk 2]
```diff
@@ -54,6 +54,7 @@
     <TargetName>dutil</TargetName>
     <MultiTargetLibrary>true</MultiTargetLibrary>
   </PropertyGroup>
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildProjectDirectory), wix.proj))\tools\WixBuild.props" />
```
Comment (robmen) [inline src/libs/dutil/dutil.vcxproj:57]: This is unfortunate. Do you know the root cause that requires us to import here? Can we fix the .targets to do the right thing?

Comment (rseanhall) [inline src/libs/dutil/dutil.vcxproj:57]: No, but balutil, deputil, and wcautil all already had this.  I thought importing .props at the beginning and .targets at the end was the standard thing to do anyway?

File: src/libs/dutil/thmutil.cpp (hunk 1 above)
Comment (rseanhall) [inline src/libs/dutil/thmutil.cpp:495]: Interesting, I didn't realize that.

File: src/libs/dutil/dutil.vcxproj (hunk 2 above)
Comment (robmen) [inline src/libs/dutil/dutil.vcxproj:57]: Should only be done when necessary, aka: a Property depends on values set in the .props file. The ideal is that there is just a single .targets import at the bottom. That removes significant complexity possibility. I'd have to look closer to see why the others do it...

File: src/libs/dutil/thmutil.cpp (hunk 1 above)
Comment (robmen) [inline src/libs/dutil/thmutil.cpp:495]: Yeah, well, not like's written down and probably not executed perfectly either. :sigh:

**Pull request merged**